stable/xray/README.md

```diff
@@ -30,7 +30,7 @@ helm repo add jfrog https://charts.jfrog.io
 ### Install Chart
 Install JFrog Xray
 ```bash
-helm upgrade --install xray --namespace xray jfrog/xray --version 2.0.2
```

Comment:
@rahulsadanandan please change to `2.1.0` here too

Reply:
Changed now